src/components/PeopleTable/PeopleTable.tsx

```diff
@@ -0,0 +1,129 @@
+import classNames from 'classnames';
+import { useEffect, useState } from 'react';
+import { useParams } from 'react-router-dom';
+import { Person } from '../../types';
+import { Loader } from '../Loader';
+import { PeopleLink } from '../PeopleLink/PeopleLink';
+import { getPeople } from '../../api';
+
+export const PeopleTable = () => {
+  const [isLoading, setIsLoading] = useState(false);
+  const [peoples, setPeoples] = useState<Person[] | []>([]);
+  const [error, setError] = useState(false);
+
+  const { selectedSlug } = useParams();
+
+  const initialLoad = async () => {
```

Comment:
Better define this function inside `useEffect`, because this way this function is recreated on each render of the component